frontend/src/concepts/pipelines/content/PipelineRunTypeLabel.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Label } from '@patternfly/react-core';
+import {
+  PipelineRunLabels,
+  getPipelineCoreResourceJobReference,
+  getPipelineCoreResourcePipelineReference,
+} from '~/concepts/pipelines/content/tables/utils';
+import { PipelineCoreResourceKF } from '~/concepts/pipelines/kfTypes';
+
+type PipelineRunTypeLabelProps = {
+  resource: PipelineCoreResourceKF;
+};
+const PipelineRunTypeLabel: React.FC<PipelineRunTypeLabelProps> = ({ resource }) => {
+  const jobReference = getPipelineCoreResourceJobReference(resource);
+  const pipelineReference = getPipelineCoreResourcePipelineReference(resource);
+
+  return (
+    <>
+      {jobReference ? (
+        <Label color="blue">{PipelineRunLabels.RECURRING}</Label>
+      ) : !pipelineReference ? (
+        <Label color="blue">{PipelineRunLabels.RECURRING}</Label>
+      ) : (
+        <Label color="blue">{PipelineRunLabels.ONEOFF}</Label>
+      )}
```

Comment:
I think we need tooltips in here -- it'll help users understand what these mean and why they are there.

Tooltips inline as comments.
```
      {jobReference ? (
        // Created by a scheduled run
        <Label color="blue">{PipelineRunLabels.RECURRING}</Label>
      ) : !pipelineReference ? (
        // Created by a scheduled run that was deleted
        <Label color="blue">{PipelineRunLabels.RECURRING}</Label>
      ) : (
        // Run once immediately after creation
        <Label color="blue">{PipelineRunLabels.ONEOFF}</Label>
      )}
```

cc @yannnz @kaedward